src/main/java/com/cozyhome/onlineshop/userservice/security/service/impl/UserServiceImpl.java

```diff
@@ -27,163 +20,174 @@
             throw new IllegalArgumentException("Invalid or expired token");
         }
 
         User user = resetToken.getUser();
         user.setPassword(encoder.encode(newPassword.getPassword()));
-		user.setModifiedAt(LocalDateTime.now());
+        user.setModifiedAt(LocalDateTime.now());
         userRepository.save(user);
 
-        securityTokenRepository.delete(resetToken);	
+        securityTokenRepository.delete(resetToken);
         return user;
-	}
-
-	@Override
-	public UserInformationResponse updateUserData(UserInformationRequest userInformationDto, String userId) {
-		User user = userRepository.findByIdAndStatus(userId, UserStatusE.ACTIVE)
-				.orElseThrow(() -> new DataNotFoundException(
-						String.format("User with email = %s not found.", userInformationDto.getEmail())));
-
-		user.setLastName(userInformationDto.getLastName());
-		user.setFirstName(userInformationDto.getFirstName());
-		user.setPhoneNumber(userInformationDto.getPhoneNumber());
-		user.setModifiedAt(LocalDateTime.now());
-
-		if (!user.getEmail().equals(userInformationDto.getEmail())) {
-			if (userRepository.existsByEmailAndStatus(userInformationDto.getEmail(), UserStatusE.ACTIVE) ) {
-				throw new DataAlreadyExistException(String.format("Email %s is already in use", userInformationDto.getEmail()));
-			}
-			user.setEmail(userInformationDto.getEmail());
-		}
-
-		if (!userInformationDto.getBirthday().isEmpty()) {
-			LocalDate birthday = LocalDate.parse(userInformationDto.getBirthday());
-			user.setBirthday(birthday);
-		}
-
-		if (!userInformationDto.getNewPassword().isEmpty()) {
-			if (encoder.matches(userInformationDto.getOldPassword(), user.getPassword())) {
-				user.setPassword(encoder.encode(userInformationDto.getNewPassword()));
-			} else {
-				throw new AuthException("Wrong old password entered.");
-			}
-		}
-
-		User updatedUser = userRepository.save(user);
-		return userBuilder.buildUserInformationResponse(updatedUser);
-	}
-
-	@Override
-	public UserInformationResponse getUserInfo(String userId) {
-		User user = userRepository.findByIdAndStatus(userId, UserStatusE.ACTIVE)
-				.orElseThrow(() -> new DataNotFoundException(String.format("User with id = %s not found.", userId)));
-		return userBuilder.buildUserInformationResponse(user);
-	}
-
-	@Override
-	public void deleteUser(String email) {
-		User user = userRepository.findByEmailAndStatus(email, UserStatusE.ACTIVE)
-				.orElseThrow(() -> new IllegalArgumentException("Not user found by the email " + email));
-		user.setModifiedAt(LocalDateTime.now());
-		user.setStatus(UserStatusE.DELETED);
-		log.info("[ON deleteUser] :: changed the user status to DELETED for the user with email {}", email);
-		userRepository.save(user);
-	}
+    }
+
+    @Override
+    public void updateUserPassword(PasswordUpdateRequest passwords, String userId) {
+        User user = userRepository.findByIdAndStatus("6525835b1578af3164f1c4d3", UserStatusE.ACTIVE)
```

Comment:
"6525835b1578af3164f1c4d3" change to userId

Reply:
done